cmake/onnxruntime_python.cmake

```diff
@@ -387,8 +387,14 @@ if (onnxruntime_ENABLE_TRAINING)
   file(GLOB onnxruntime_python_ortmodule_torch_cpp_ext_fused_ops_srcs CONFIGURE_DEPENDS
     "${ORTTRAINING_SOURCE_DIR}/python/training/ortmodule/torch_cpp_extensions/cuda/fused_ops/*"
   )
+  file(GLOB onnxruntime_python_ort_triton_srcs CONFIGURE_DEPENDS
```

Comment:
include these only if triton is enabled?

Reply:
Tried but caused problem for amd package. Keep this way as same as torch interop for now.